.devcontainer/scripts/notify-dev-entrypoint.sh

```diff
@@ -33,7 +36,7 @@ echo -e "complete -F __start_kubectl k" >> ~/.zshrc
 cd /workspaces/notification-manifests
 
 # kubent
- sh -c "$(curl -sSL https://git.io/install-kubent)"
+sh -c "$(curl -sSL https://git.io/install-kubent)"
 
-# Warm up git index prior to display status in prompt
-git status
+# Explicitly mark workspace as a safe git directory
+git config --global --add safe.directory /workspaces/notification-manifests
```

Comment:
On the other hand, this should be useful since we need to approve the git workspace folder every time we rebuild the container or open on a new git folder in general.